src/styles/main.scss

```diff
@@ -0,0 +1,99 @@
+@import "./variables.scss";
+@import "./animation.scss";
+
+body {
+  margin: 0;
+}
+
+.clock {
+  display: block;
+  position: relative;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: $margin-value;
+  margin-bottom: $margin-value;
```

Comment:
The checklist 
<img width="1134" alt="image" src="https://user-images.githubusercontent.com/99403198/209121512-64acfa3d-1392-4565-82bd-b7c654e59b90.png">

Reply:
Probably I am not understanding something, could you explain to me please?

Reply:
You have bad and good examples 
![image](https://user-images.githubusercontent.com/70009205/209369600-03a3634d-cd06-46f0-bce7-d36dab570829.png)
do you read [checklist](https://github.com/mate-academy/layout_stop-watch/blob/master/checklist.md)?

You should create extra class for this div and another for main to show that your div is an element and not a block. If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

Reply:
Oh, sorry, sure! I read check list, and it's clear to me in example, but I was not sure where I should apply this rule in my case( ( I also understood about fe_chat